### ISSUE TITLE
TensorRT 8.4.0.6  3D deconvolution undefined symbol: getBuilderPluginRegistry

### ISSUE THREAD

**Issue by Zero-Qzy** [NVIDIA/TensorRT#1987]
## Description

Hello， I use the TensorRT to transform the model and this problem occurs:
```
Traceback (most recent call last):
  File "onnx2trt.py", line 3, in <module>
    import tensorrt as trt
  File "/home/amax/anaconda3/lib/python3.8/site-packages/tensorrt/__init__.py", line 67, in <module>
    from .tensorrt import *
ImportError: /home/amax/anaconda3/lib/python3.8/site-packages/tensorrt/tensorrt.so: undefined symbol: getBuilderPluginRegistry
```

part of my source code are follows:
```
    self.up_1 = nn.ConvTranspose3d(in_channels=64, out_channels=32, kernel_size=3, stride=2, padding=1,output_padding=1, bias=True)
    self.vrn2_1 = vrn_block(num_filters=32, vrnin_chs=32)
    self.vrn2_2 = vrn_block(num_filters=32, vrnin_chs=32)
    self.vrn2_3 = vrn_block(num_filters=32, vrnin_chs=32)
```
vrn is make up of nn.Conv3d:
```

    self.conv1_1 = nn.Conv3d(in_channels=in_chs, out_channels=int(num_filters/4),kernel_size=3, stride=1, padding=1, bias=True)
    self.conv1_2 = nn.Conv3d(in_channels=int(num_filters/4), out_channels=int(num_filters/2),kernel_size=3, stride=1, padding=1, bias=True)

    self.conv2_1 = nn.Conv3d(in_channels=in_chs, out_channels=int(num_filters/4),kernel_size=1, stride=1, padding=0, bias=True)
    self.conv2_2 = nn.Conv3d(in_channels=int(num_filters/4), out_channels=int(num_filters/4),kernel_size=3, stride=1, padding=1, bias=True)
    self.conv2_3 = nn.Conv3d(in_channels=int(num_filters/4), out_channels=int(num_filters/2),kernel_size=1, stride=1, padding=0, bias=True)
```

## Environment

**TensorRT Version**: 8.4.0.6
**NVIDIA GPU**: RTX A5000 
**NVIDIA Driver Version**: 495.44
**CUDA Version**: 11.5
**CUDNN Version**: 
**Operating System**: ubuntu 20.04
**Python Version (if applicable)**: 3.8.8
**Tensorflow Version (if applicable)**: None
**PyTorch Version (if applicable)**: 1.10.0
**Baremetal or Container (if so, version)**: 

How can I solve this problem？

Thank you for your help!

**Comment by Zero-Qzy** [comment]
my transform code is:
```python
def get_engine():

    logger = trt.Logger(trt.Logger.ERROR)
    if os.path.isfile(trtFile):
        with open(trtFile, 'rb') as f:
            engine = trt.Runtime(logger).deserialize_cuda_engine(f.read())
        if engine == None:
            print("Failed loading engine!")
            exit()
        print("Succeeded loading engine!")

    else:
        builder = trt.Builder(logger)
        network = builder.create_network(1 << int(trt.NetworkDefinitionCreationFlag.EXPLICIT_BATCH))
        config = builder.create_builder_config()
        config.max_workspace_size = 3 << 30


      
        # config = builder.create_builder_config()
        # config.max_workspace_size = 3 << 30

        parser = trt.OnnxParser(network, logger)
        if not os.path.exists(onnxFile):
            print("Failed finding onnx file!")
            exit()
        print("Succeeded finding onnx file!")

        with open(onnxFile, 'rb') as model:
            if not parser.parse(model.read()):
                print("Failed parsing .onnx file!")
                for error in range(parser.num_errors):
                    print(parser.get_error(error))
                exit()
            print("Succeeded parsing .onnx file!")

        engine = builder.build_cuda_engine(network, config)

        with open(trtFile, 'wb') as f:
            f.write(engine.serialize())
        return engine
```

**Comment by nvpohanh** [comment]
@Zero-Qzy Could you make sure that `libnvinfer_plugin.so` is in your LD_LIBRARY_PATH paths?

**Comment by Zero-Qzy** [comment]
@nvpohanh Hi,there are so many versions of TensorRT in my environment. I change the order of my LD_LIBRARY_PATH and LIBRARY_PATH, so the problem is solved. Thank you for your help!

**Comment by nvpohanh** [comment]
Got it! closing this

**Comment by YOOHYOJEONG** [comment]
@Zero-Qzy Hi, May I ask how you solved this problem?
I faced the same error as you.

Looking at your answer, I think "LD_LIBRARY_PATH" is the problem, but how did you modify "LD_LIBRARY_PATH"?
My "LD_LIBRARY_PATH" appears as "/usr/local/cuda-10.2/lib64". How should I fix it?

I've already finished building and installing Tensorrt. I'm so sad because I got this error even though I only have to import it.
